src/Tribe/RSVP.php

```diff
@@ -1370,6 +1370,12 @@ public function send_tickets_email( $order_id, $event_id = null ) {
 		if ( $sent ) {
 			foreach ( $all_attendees as $attendee ) {
 				$this->update_ticket_sent_counter( $attendee['qr_ticket_id'], $this->attendee_ticket_sent );
+
+				$this->update_attendee_activity_log( $attendee['attendee_id'], [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 678274ddad979dc01e99655e97cc2572</sub>